docs/list.md

```diff
@@ -223,6 +223,14 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
     - [Gain points via Code contributions](https://github.com/formbricks/formbricks/issues)
     - [Gain points via Social Media/Design/Documentation](https://formbricks.notion.site/FormTribe-Side-Quests-4ab3b294cfa04e94b77dfddd66378ea2)
 
+#### **Featureform**
+
+- **Swag**: Accepted PRs will get a Featureform Swag package and will be entered for a raffle for a $500 gift card
+- **Requirements**: One merged PR or tutorial submission.
```

Comment:
Can you please be more briefly specific about the requirements based on the [Contribution Instructions](https://github.com/featureform/hacktoberfest/blob/main/README.md)?